content/en/docs/collector/howto-custom-collector.md

```diff
@@ -0,0 +1,115 @@
+
+```
+go install go.opentelemetry.io/collector/cmd/builder@latest
+```
+
+The `builder` command will be installed on your `$GOPATH/bin`, which most likely is already added to your path.
+
+To make sure it's there, go to your terminal and type `builder help`, and once you hit enter you should have the following output:
+
+```
+
+OpenTelemetry Collector distribution builder (dev)
+
+Usage:
+  builder [flags]
+  builder [command]
+
+Available Commands:
+  completion  Generate the autocompletion script for the specified shell
+  help        Help about any command
+  version     Version of opentelemetry-collector-builder
+
+Flags:
+      --config string            config file (default is $HOME/.otelcol-builder.yaml)
+      --description string       A descriptive name for the OpenTelemetry Collector distribution (default "Custom OpenTelemetry Collector distribution")
+      --go string                The Go binary to use during the compilation phase. Default: go from the PATH
+  -h, --help                     help for builder
+      --module string            The Go module for the new distribution (default "go.opentelemetry.io/collector/cmd/builder")
+      --name string              The executable name for the OpenTelemetry Collector distribution (default "otelcol-custom")
+      --otelcol-version string   Which version of OpenTelemetry Collector to use as base (default "0.42.0")
+      --output-path string       Where to write the resulting files (default "/var/folders/86/s7l1czb16g124tng0d7wyrtw0000gn/T/otelcol-distribution3618633831")
+      --skip-compilation         Whether builder should only generate go code with no compile of the collector (default false)
+      --version string           The version for the OpenTelemetry Collector distribution (default "1.0.0")
+
+Use "builder [command] --help" for more information about a command.
+```
+
+## Step 2 - Create a builder config file
+
+The builder config file is a `yaml` where you basically pass information about the code generation and compile process combined with the components that you would like to add to your Collector's distribution.
+
+Go ahead and create a `builder-config.yaml` with the following content:
+
+> builder-config.yaml
+```yaml
+dist:
+    module: github.com/open-telemetry/opentelemetry-collector # the module name for the new distribution, following Go mod conventions. Optional, but recommended.
+    name: otelcol-mybuild # the binary name. Optional.
+    description: "Custom OpenTelemetry Collector distribution" # a long name for the application. Optional.
+    otelcol_version: "0.41.0" # the OpenTelemetry Collector version to use as base for the distribution. Optional.
+    output_path: ./otelcol-mybuild # the path to write the output (sources and binary). Optional.
+    version: "1.0.0" # the version for your custom OpenTelemetry Collector. Optional.
+    go: "/usr/local/go/bin/go" # which Go binary to use to compile the generated sources. Optional.
```

Comment:
Same here.

Reply:
which setting?

Reply:
`go`

Reply:
fixed in: https://github.com/open-telemetry/opentelemetry.io/pull/1277#:~:text=to%20support%20suggestions.-,fd2e9a3,-Merge%20branch%20%27doc